cerberus/validator.py

```diff
@@ -666,8 +666,8 @@ def __normalize_sequence(self, field, mapping, schema):
             document_crumb=field, schema_crumb=(field, 'schema'),
             schema=schema)
         result = validator.normalized(document, always_return_document=True)
-        for i in result:
-            mapping[field][i] = result[i]
+        if result:
```

Comment:
You are totally right. Sorry about that, and thank you for your guide and your time. 😄